### PR TITLE
Change image status crd

### PR DESCRIPTION
**Pull request by ashnamehrotra**
change apis to report statuses by node in ImageStatus and ImageList

### CODE REVIEW

File: config/samples/eraser_v1alpha1_imagestatus.yaml [hunk 1]
```diff
@@ -3,16 +3,14 @@ kind: ImageStatus
 metadata:
   name: imagestatus-sample
 status:
-  timestamp: "2020-05-11T01:46:13Z"
+  node: "aks-nodepool1-27292825-vmss000000"
```
Comment (sozercan) [inline config/samples/eraser_v1alpha1_imagestatus.yaml:6]: looks like timestamp is gone

File: api/v1alpha1/imagestatus_types.go [hunk 2]
```diff
@@ -48,16 +44,18 @@ type ImageStatus struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Status ImageStatusStatus `json:"status,omitempty"`
+	Spec ImageStatusSpec `json:"spec,omitempty"`
```
Comment (sozercan) [inline api/v1alpha1/imagestatus_types.go]: let's remove this after testing is done

**Review by sozercan — APPROVED** [review]
LGTM

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**